flexmeasures/cli/data_add.py

```diff
@@ -174,6 +174,11 @@ def new_account_role(name: str, description: str):
     callback=validate_color_hex,
     help="Secondary color to use in UI, in hex format. Defaults to FlexMeasures' secondary color (#f1a122)",
 )
+@click.option(
+    "--logo-url",
+    callback=validate_url,
```

Comment:
The db revision file suggests the max length is 255 characters. Let's add validation for that.

Also, maybe we should demand shorter urls.

Reply:
Hmm, ok will work on that